redesign/index.html

```diff
@@ -2,9 +2,207 @@
+      <p>We work together to understand the problem
+        that has been identified and confirm the intended users feel the pain
+        and need for a solution.</p>
+      </div>
+      <div class="w5 w-30-ns mv3 mh1 pa2 dwyl-dark-grey dib-ns tl dtc-ns v-top center">
+        <h3 class="br-100 w2 h2 tc dtc v-mid bg-yellow">2</h3>
+        <h3 class="fw5 f4">Make the next market-ready version of the solution</h3>
+        <p>We build the solution following agile/lean
+          software development best practices and get feedback from users as fast
+          as possible.</p>
+      </div>
+      <div class="w5 w-30-ns mv3 mh1 pa2 dwyl-dark-grey dib-ns tl dtc-ns v-top center">
+        <h3 class="br-100 w2 h2 tc dtc v-mid bg-yellow">3</h3>
+        <h3 class="fw5 f4">Review the feedback and return to step 1</h3>
+        <p>We review all feedback and confirm we are
+          making progress toward solving the problem.</p>
+      </div>
+    </div>
+
+    <h2 class="dwyl-mint w-80 w-40-ns pl3-ns dib dtc-ns v-top">Meet your needs quickly and without waste.</h2>
+    <h3 class="dwyl-dark-grey w-70 w-50-ns pl3-ns dib b dtc-ns v-top">By iterating over these 3 simple steps we
+      maintain a tight feedback loop betweem the people building and the people
+      using.</h3>
+    <div class="tc mt4-ns pl6-ns pb5">
+      <a href="#" class="dib link mv2 pa3 pa3-ns dwyl-bg-mint white br2 f4 shadow-4">How Can We Help?</a>
+    </div>
+    </section>
+
+    <section class="w-100 tl">
+      <div class="dwyl-bg-mint white">
+        <h1 class="pv3 text-shadow f1 mv0 tc">Training <span class="sr-only">and</span>+ Workshops</h1>
+      </div>
+      <div class="pt2 pl3 pl6-ns dwyl-dark-grey">
+        <h3 class="fw9 f3 w-80 w-100-ns center">Hands-on learning to Empower and Inspire.</h3>
+        <p class="w-80 w-100-ns center">Join one of our masterclasses or tailor a class to suit your needs</p>
+        <ul class="b pl4">
+          <div class="dib-ns">
+            <li>Agile Development Methodology</li>
+            <li>Customer Development</li>
+            <li>Cyber Security</li>
+          </div>
+          <div class="dib-ns pl5-ns">
+            <li>User Testing</li>
+            <li>Product Ownership</li>
+            <li>Advanced Analytics</li>
+          </div>
+          <div class="dib-ns pl5-ns">
+            <li>Human-centered Design</li>
+            <li>Code</li>
+            <li>Data Visualisation</li>
+          </div>
+        </ul>
+      </div>
+      <a href="#" class="dib link mt2 mb4 ml4 ml3-ns ml6-ns ph4 pv2 dwyl-bg-mint white br2 f5 shadow-4">Enquire Now</a>
+    </section>
+
+    <section class="dwyl-bg-mint center pt4 pb5-ns">
+
+      <div class="dib-ns w-50-ns dtc v-top">
+        <div class="w-70 w-40-ns white center">
+          <h1 class="mt0-ns mb3 pl4-ns f-xl tl fw9">Let's</h1>
+          <h1 class="pl4-ns mt3 f-xl tl fw9">Talk</h1>
+        </div>
+        <div class="w-80-ns w-90 ml3">
```

Comment:
https://www.elegantthemes.com/blog/tips-tricks/telephone-links-how-to-add-call-able-links-ctas-to-your-website